chronos/internal/asyncfutures.nim

```diff
@@ -1667,8 +1667,7 @@ proc `or`*[T, Y, E1, E2](
   type
     InternalRaisesFutureRaises = union(E1, E2)
 
-  let
-    retFuture = newFuture[void]("chronos.wait()", {FutureFlag.OwnCancelSchedule})
+  let retFuture = newFuture[void]("chronos.or()", {})
```

Comment:
`cancelSoon(fut or fut2)` will defect in this case?

Reply:
why?

Reply:
nevermind, my mistake